site2/docs/reference-configuration.md

```diff
@@ -598,6 +598,7 @@ You can set the log level and configuration in the  [log4j2.yaml](https://github
 | tokenAudience | The token audience stands for this broker. The field `tokenAudienceClaim` of a valid token need contains this parameter.| |
 |saslJaasClientAllowedIds|This is a regexp, which limits the range of possible ids which can connect to the Broker using SASL. By default, it is set to `SaslConstants.JAAS_CLIENT_ALLOWED_IDS_DEFAULT`, which is ".*pulsar.*", so only clients whose id contains 'pulsar' are allowed to connect.|N/A|
 |saslJaasServerSectionName|Service Principal, for login context name. By default, it is set to `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "Broker".|N/A|
```

Comment:
This default value is different from the one shown in the `security-kerberos` file, which is "PulsarBroker". Which one is correct?

Reply:
PulsarBroker is correct